### ISSUE TITLE
If user does not pass a composition in the command line, show a list of compositions

### ISSUE THREAD

**Issue by JonnyBurger** [remotion-dev/remotion#1377]
[<img alt="Issuehunt badges" src="https://img.shields.io/badge/IssueHunt-%240%20Funded-%2300A156.svg" />](https://issuehunt.io/r/remotion-dev/remotion/issues/1377)

**🎃 This issue is part of our Hacktoberfest campaign!**
📚 [Read more about Hacktoberfest here](https://remotion.dev/hacktoberfest)
✅ This issue is currently taken by @Uzunov-Stefan! 
💰 Thanks to [CodeChem](https://codechem.com) for sponsoring this issue!

There are several commands that take a composition:

```bash
npx remotion render src/index.tsx [composition-name]
npx remotion still src/index.tsx [composition-name]
npx remotion benchmark src/index.tsx [composition-name]
```

The goal is that one can omit the composition name:

```bash
npx remotion render src/index.tsx
npx remotion still src/index.tsx
npx remotion benchmark src/index.tsx
```

## Acceptance criteria

- Create a generic solution that asks the user what compositions they want to render when they don't specify the composition name
- The logic from here can be copy-pasted and the `prompts` module or another lightweight module may be used: https://github.com/uragirii/remotion/blob/6323e86f076f6b7979a1beb8c1bcf92964e583e8/packages/create-video/src/prompts.ts#L3
- Don't ask and fail if running on a CI `process.env.CI`
- Use getCompositions() to get the list of compositions (afaik this is already done)
- If there is only 1 compositions, choose that one
- Edge case handling: Normally you can specify an output name `npx remotion render src/index.tsx [composition-name] [out-name]`. If you don't include the composition name, you cannot set the out name either and the default will be used.

---

<details>
<summary>
<b>IssueHunt Summary</b>
</summary>

### Tips

- Checkout the [Issuehunt explorer](https://issuehunt.io/r/remotion-dev/remotion/) to discover more funded issues.
- Need some help from other developers? [Add your repositories](https://issuehunt.io/r/new) on IssueHunt to raise funds.
</details>

**Comment by clearlysid** [comment]
oh yes please, this feature will be quite a timesaver for me. @JonnyBurger I'm happy to help fund this too, if you'd like.

**Comment by JonnyBurger** [comment]
@clearlysid Thanks a lot! No need, we got you 😁

**Comment by Uzunov-Stefan** [comment]
Hi @JonnyBurger 
I would like to work on this issue

**Comment by JonnyBurger** [comment]
@Uzunov-Stefan Reserved it for you! No guarantee that this will become a Hacktoberfest issue or that it will hold a bounty, but stay tuned because we hope to make it happen 🙂

**Comment by JonnyBurger** [comment]
@Uzunov-Stefan The issue now has a bounty for Hacktoberfest! Would you stilll like to work on this issue?

**Comment by Uzunov-Stefan** [comment]
@JonnyBurger 
Yes, I'd still like to work on this issue
Thanks

**Comment by JonnyBurger** [comment]
@Uznov-Stefan perfect!

**Comment by JonnyBurger** [comment]
@Uzunov-Stefan has said he wants to waive the bounty and to be reassigned to a different issue. We are happy to do that!

**Comment by JonnyBurger** [comment]
@Uzunov-Stefan Are you still active on this issue? If not, let me know and I'll reassign it to someone else!

**Comment by Uzunov-Stefan** [comment]
Yes I am, should be done by tomorrow :rocket:

**Comment by JonnyBurger** [comment]
@Uzunov-Stefan Awesome, sorry for asking again. There is no rush :)

**Comment by JonnyBurger** [comment]
Fixed by #1438
